apps/frontend/src/libs/modules/map-client/libs/types/types.ts

```diff
@@ -0,0 +1,53 @@
+import { type PointGeometry } from "@smartscapes/shared";
+import { type default as mapboxgl } from "mapbox-gl";
+
+type AccessToken = MapOptions["accessToken"];
+
+type ControlPosition = mapboxgl.ControlPosition;
+
+type GeolocateControlOptions = mapboxgl.GeolocateControlOptions & {
```

Comment:
1. Let's define all the properties explicitly, it's ok use mapboxgl.GeolocateControlOptions type for internal implementation, but if we are going to allow passing options in such a format from the outside to other components, that's not what we want. So we either need to have a separate type for internal config and external config, or just let's define only properties that are used right now. And the same for other types here that extend mapbox types
2. Haven't noticed this previous time, but we also need to split all types into the respective files - name.type.ts
```suggestion
type GeolocateControlOptions = {
```